content/en/user-manual/graphics/shader-chunk-migrations.md

```diff
@@ -27,6 +27,33 @@ By doing this you will no longer see warning messages in the console.
 ## Chunk changes
 
 The following tables break down the chunk changes by Engine release.
+
+---
+#### *Engine v1.57*
+In 1.57, almost all front-end chunks have been changed to minimize the amount of samplers used by the shader. This is an optional feature, however it's recommended to follow the same coding style to reduce the amount of samplers used by the shader. 
+
+| Chunk | Changes |
+| ---   | ---     |
+| `aoPS` | <ul><li>sampler declaration removed and texture is sampled from the $SAMPLER identifier</li></ul> |
```

Comment:
It might make sense to have a single row here with all the chunks listed on the left and a single change description?